src/stepper/stepperComponent.tsx

```diff
@@ -0,0 +1,268 @@
+import React, { useRef, useEffect, useState } from "react";
```

Comment:
can we split this file into multiple for each component used. I feel like the file is pretty long